pkg/sql/sqlbase/testutils.go

```diff
@@ -689,13 +702,13 @@ var (
 )
 
 var (
-	// seedTypes includes the following types that form the basis of randomly
+	// SeedTypes includes the following types that form the basis of randomly
```

Comment:
IDK how I feel about exporting these various slices + helper functions, but it doesn't seem like there's another way to incorporate a dynamic set of types that the `smither` manages into these random type functions.

Reply:
I love exporting this kind of stuff. I've wanted the randInterestingDatums to be exported before too. Export everything in testutils is +1 from me.

Reply:
sounds good!